This patch is artificially crafted to have the smallest possible
dependence on patched source.

Index: configure
===================================================================
--- configure	(revision 118066)
+++ configure	(working copy)
@@ -916,4 +916,5 @@
 target_libraries="target-libiberty \
+		target-libga60 \
Index: Makefile.def
===================================================================
--- Makefile.def	(revision 118066)
+++ Makefile.def	(working copy)
@@ -115,1 +115,2 @@
 target_modules = { module= libgfortran; };
+target_modules = { module= libga60; };
